src/test/java/duke/command/DeadlineTest.java

```diff
@@ -0,0 +1,88 @@
+package duke.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeCommandException;
+import duke.exception.DukeStorageException;
+
+public class DeadlineTest {
+
+    private String path = java.nio.file.Paths.get("src", "test").toString();
+
+    @Test
+    public void test1() {
```

Comment:
The test method name can be more descriptive for readers to understand it better? There's the three part format featureUnderTest_testScenario_expectedBehavior() that you can consider adopting:)